src/main/webapp/inc.modul.jsp

```diff
@@ -1,1772 +1,1470 @@
-									+ rs.getString("e.ID")
-									+ " AND ehe.EditorID=s.ID");
-					boolean first = true;
-					while (rs2.next()) {
-						if (!first)
-							out.print(" / ");
-						out.print(rs2.getString("s.Bezeichnung"));
-						first = false;
-					}
-					out.println("</td>");
-					out.println("</tr>");
-				}
-				if(firstEdition)out.println("<td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>");
-					count++;
-					if (count % 2 == 0)
-						out.println("<tr>");
-					else
-						out.println("<tr bgcolor='#AACCDD'>");
+
+    try {
+
+        List<Object[]> resultList = ModulIncDB.getListGastQuelleEditionen("1", id);
+
+        for (Object[] row : resultList) {
+
+            if (!firstEdition) {
+                count++;
+                if (count % 2 == 0) {
+                    out.println("<tr>");
+                } else {
+                    out.println("<tr bgcolor='#AACCDD'>");
+                }
+                out.println("<td></td>");
+            }
+            firstEdition = false;
+
+            String eId = row[0] != null ? String.valueOf(row[0]) : "-1";
+            String titel = row[1] != null ? String.valueOf(row[1]) : "";
+            String reihe = row[2] != null ? String.valueOf(row[2]) : "--";
+            String band = row[3] != null ? String.valueOf(row[3]) : "--";
+            String ort = row[4] != null ? String.valueOf(row[4]) : "--";
+            String jahr = row[5] != null ? String.valueOf(row[5]) : "--";
+            String qiSeiten = row[6] != null ? String.valueOf(row[6]) : row[7] != null ? String.valueOf(row[7]) : "--";
+            String nummer = (row[8] != null && !String.valueOf(row[8]).trim().equals("")) ? String.valueOf(row[8]) : "--";
+
+            out.println("<td>" + titel + "</td>");
+            out.println("<td>" + DBtoHTML(reihe) + "</td>");
+            out.println("<td>" + DBtoHTML(band) + "</td>");
+            out.println("<td>" + DBtoHTML(ort) + "</td>");
+            out.println("<td>" + DBtoHTML(jahr) + "</td>");
+            out.println("<td>" + DBtoHTML(qiSeiten) + "</td>");
+            out.println("<td>" + DBtoHTML(nummer) + "</td>");
+
+            if (nummer != null && !nummer.trim().equals("")) {
+                showNummer = true;
+            }
+
+            out.println("<td>");
+
+            List<String> resultList_2 = ModulIncDB.getListGastQuelleEditionHerausgeber(eId);
+            boolean first = true;
+
+            for (String row_2 : resultList_2) {
+
+                String bezeichnung = row_2 != null ? row_2 : "";
+
+                if (!first) {
+                    out.print(" / ");
+                }
+                out.print(bezeichnung);
+                first = false;
+            } //while 2 ende
+            out.println("</td>");
+            out.println("</tr>");
+        } //while 1 ende
+
+        if (firstEdition) {
+            out.println("<td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>");
```

Comment:
I know it has been like this before, but maybe this would be a good opportunity to optimize:
`<td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td>`
is equivalent to
`<td colspan="8"></td>`

Reply:
done